configure.ac

```diff
@@ -6773,11 +6773,11 @@ AM_CONDITIONAL([BUILD_SHA512],[test "x$ENABLED_SHA512" = "xyes" || test "x$ENABL
 AM_CONDITIONAL([BUILD_DSA],[test "x$ENABLED_DSA" = "xyes" || test "x$ENABLED_USERSETTINGS" = "xyes"])
 AM_CONDITIONAL([BUILD_ECC],[test "x$ENABLED_ECC" != "xno" || test "x$ENABLED_USERSETTINGS" = "xyes"])
 AM_CONDITIONAL([BUILD_ED25519],[test "x$ENABLED_ED25519" = "xyes" || test "x$ENABLED_USERSETTINGS" = "xyes"])
-AM_CONDITIONAL([BUILD_ED25519_SMALL],[test "x$ENABLED_ED25519_SMALL" = "xyes" || test "x$ENABLED_USERSETTINGS" = "xyes"])
```

Comment:
Why remove the user settings check on these?

Reply:
When `ENABLED_USERSETTINGS` is on, we only build `ENABLED_ED25519_SMALL`. When `user_settings.h` only contains `HAVE_ED25519` without `ED25519_SMALL` then this causes compilation errors. Really, both `ge_operations.c` and `ge_low_mem.c` should get included in the files to build in `src/include.am` so that `user_settings.h` can choose which one is the desired version.